examples/onft721-zksync/layerzero.config.ts

```diff
@@ -0,0 +1,85 @@
+import { EndpointId } from '@layerzerolabs/lz-definitions'
+import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities'
+
+import type { OAppEdgeConfig, OAppOmniGraphHardhat, OmniPointHardhat } from '@layerzerolabs/toolbox-hardhat'
+
+const abstractContract: OmniPointHardhat = {
+    eid: EndpointId.ABSTRACT_V2_TESTNET,
+    contractName: 'MyONFT721',
+}
+
+const fujiContract: OmniPointHardhat = {
+    eid: EndpointId.AVALANCHE_V2_TESTNET,
+    contractName: 'MyONFT721',
+}
+
+const zksyncContract: OmniPointHardhat = {
+    eid: EndpointId.ZKSYNCSEP_V2_TESTNET,
+    contractName: 'MyONFT721',
+}
+
+const DEFAULT_EDGE_CONFIG: OAppEdgeConfig = {
+    enforcedOptions: [
+        {
+            msgType: 1,
+            optionType: ExecutorOptionType.LZ_RECEIVE,
+            gas: 100_000,
```

Comment:
Kind of a NIT, but can you just add a comment saying `gas` should be profiled later?  I know our docs aren't amazing on this yet, but we will work to improve them.  I just want folks to know that they can go < 100K if it fits the deployment situation.